qdrant-landing/content/blog/superlinked-multimodal-search.md

```diff
@@ -0,0 +1,271 @@
+
+The result is a search experience that feels intuitive and "just works" - whether users are looking for "pet-friendly boutique hotels with good reviews near the city center" or "affordable family suites with pool access in resort areas" - because the system understands both the semantics and the relationships between different attributes of what users are asking for.
+
+> As search expectations continue to evolve, this multimodal and multidimensional hybrid search approach represents not just an incremental improvement but a fundamental rethinking of how search should work. 
+
+The hotel search demo showcases this vision in action, a glimpse into a future where search understands not just the words we use, but the complex, nuanced preferences they represent.
+
+## How to Build the App
+
+For more details, [check out the documentation](https://github.com/superlinked/hotel-search-recipe-qdrant).
+
+Otherwise, you can clone the app:
+
+```shell
+git clone https://github.com/superlinked/hotel-search-recipe-qdrant.git
+```
+
+The backend is located under `superlinked_app`, while the frontend has to be built from `frontend_app`.
+
+### Deploy the Backend
+
+Use [`superlinked_app/.env-example`](./superlinked_app/.env-example) as a template, create `superlinked_app/.env` and set `OPENAI_API_KEY` required for Natural Query Interface, `QDRANT_URL` and `QDRANT_API_KEY` required for Qdrant Vector Database.
+
+```shell
+python3.11 -m venv .venv
+. .venv/bin/activate
+pip install -r requirements.txt
+APP_MODULE_PATH=superlinked_app python -m superlinked.server
+```
+
+It will take some time (depending on the network) to download the sentence-transformers model for the very first time.
+
+API docs will be available at [localhost:8080/docs](http://localhost:8080/docs).
+
+To ingest the dataset, run this command in your terminal:
+```shell
+curl -X 'POST' \
+  'http://localhost:8080/data-loader/hotel/run' \
+  -H 'accept: application/json' \
+  -d ''
+```
+Please wait until the ingestion is finished. You will see the message.
+
+#### Inspecting Collections in Qdrant Cloud Dashboard
+
+Once your Superlinked vectors are ingested, log in to the Qdrant Cloud dashboard to navigate to **Collections** and select your `default` hotel collection.
+
+![default-collection](/blog/superlinked-multimodal-search/default-collection.png)
+
+You can browse individual points under the **Data** tab to view payload metadata (price, rating, amenities) alongside their raw vector embeddings.
+
+![collection-information](/blog/superlinked-multimodal-search/collection-information.png)
+
+In the **Collection Infromation** section, you can use the **Search** tab to run real-time KNN queries or apply metadata filters. In the **Search Quality** section, you can also monitor performance metrics (throughput, latency).
```

Comment:
Do we really have a Search tab in the collection info section? I checked one of my clusters, and I couldn't find anything like this. If you mean the search bar, then it's not KNN, but more of a filtering mechanism. For nearest neighbour search, we would need to provide vectors.